app/services/history_recorder.rb

```diff
@@ -4,7 +4,9 @@ def initialize(user:)
   end
 
   def call(changes:, reference:, activity:, report: nil)
-    changes.each do |value_changed, (previous_value, new_value)|
+    changes
+      .delete_if { |value_changed, _| value_changed == "form_state" }
+      .each do |value_changed, (previous_value, new_value)|
```

Comment:
Would it not be clearer if we called `next` if the `value_changed` was `form_state`. Might be good if we had `"form_state"` in an array constant called something like `INTERNAL_VALUES` to make it clear why we're skipping this particular variable (we may also want to add others)

Reply:
yes, both excellent suggestions, thank you!